x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/common_fields/properties_field.tsx

```diff
@@ -29,10 +29,10 @@ const fieldsConfig: FieldsConfig = {
 
 interface Props {
   helpText?: React.ReactNode;
-  propertyOptions?: EuiComboBoxOptionOption[];
+  euiFieldProps?: EuiComboBoxProps<string>;
```

Comment:
I changed this one a little bit to just accept euiFieldProps instead, so that we can pass through other props we might require and not just the options. This component is only used for `UserAgent` and `GeoIP` processors.

Reply:
Geoip processor doesnt take any extra props, so no changes needed there !